src/CMakeLists.txt

```diff
@@ -448,6 +448,8 @@ dbms_target_link_libraries(PRIVATE ch_contrib::zstd)
 target_link_libraries (clickhouse_common_io PUBLIC ch_contrib::zstd)
 target_link_libraries (clickhouse_common_io PUBLIC ch_contrib::xz)
 
+target_link_libraries(clickhouse_common_io PUBLIC ch_contrib::rocksdb)
```

Comment:
Why?

Reply:
We have used the function crc32c_ppc from contrib/rocksdb/util/crc32c_ppc.c. Linked the library to common to avoid the linking error.